tests/fixtures/setup/setup_tahoma_1.json

```diff
@@ -0,0 +1,7372 @@
+{
+    "creationTime": 1501224054000,
+    "lastUpdateTime": 1501224054000,
+    "id": "SETUP-1234-1234-6233",
+    "location": {
+        "creationTime": 1501224054000,
+        "lastUpdateTime": 1536691223000,
+        "city": "* * *",
+        "country": "*",
+        "postalCode": "*",
+        "addressLine1": "*, * * *",
+        "addressLine2": "",
+        "timezone": "Europe/Paris",
+        "longitude": "*",
+        "latitude": "*",
+        "twilightMode": 2,
+        "twilightAngle": "CIVIL",
```

Comment:
This payload was just copied from `setup_tahoma_3.json` with the only change being the removal of `twilightCity`.

Not sure yet how to properly capture an actual Tahoma v1 payload.

Reply:
Thx for adding test. We haven’t yet a fixture for Tahoma v1 indeed. You can use you own one:
 * Go to tahomalink.com 
 * Enable the Inspector
 * Open the network tab filtering on the word `setup`
 * Log in
 * You will see a request `setup?_=xxxx`
 * Copy the content of the response
 * Paste it within setup_tahoma_1.json
 * Finally run this [script](https://github.com/iMicknl/python-overkiz-api/blob/main/utils/mask_fixtures.py) to hide sensitive data.

Reply:
Done!